realm/realm-library/src/objectServer/java/io/realm/PermissionManager.java

```diff
@@ -299,17 +299,21 @@ public RealmAsyncTask acceptOffer(String offerToken, final AcceptOfferCallback c
     }
 
     /**
-     * FIXME
-     * @param offerToken
-     * @return
+     * Revokes an existing offer. This will prevent any other users from accepting it. Users that already accepted it,
+     * will not be affected.
+     *
```

Comment:
Technically it will not happen until we uploaded the deletion to the server. I added a description about this + made sure the callback isn't invoked until we know this has been done.